apps/clinic/src/modules/doctors/dtos/doctor-query.dto.ts

```diff
@@ -1,6 +1,19 @@
 import { ApiProperty } from '@nestjs/swagger';
 
+// Find way how to import it from SelectQueryBuilder orderBys.direction
+export type QueryBuilderSortDirection = 'ASC' | 'DESC';
+
 export class DoctorQueryDto {
   @ApiProperty({ required: false })
   search: string;
+
```

Comment:
based on the repository getDoctors i've noticed that search = specialization, so it should be named as such, "search" is very vague